mediapipe/tasks/ios/vision/image_classifier/sources/MPPImageClassifierOptions.h

```diff
@@ -31,6 +31,7 @@ NS_SWIFT_NAME(ImageClassifierOptions)
 /**
  * The user-defined result callback for processing live stream data. The result callback should only
  * be specified when the running mode is set to the live stream mode.
+ * TODO: Add parameter `MPPImage` in the callback.
```

Comment:
Can we overwrite the `dealloc` method on MPPImage to achieve this? We should really not be creating a copy, and we are updating the Java code to also avoid this copy.

Reply:
If we are not creating a copy, we don't need to do anything in dealloc. I am assuming MediaPipe::Image will handle the deallocation of the buffer? We will simply pass the reference to the CVPixelBuffer. The pixel buffer itself will get deallocated when it goes out of scope by ARC. But if the reference to the uint8 buffer is not deallocated before the dealloc is called, then pixel buffer may not be de allocated. So I need to test this out properly. 
I am trying to avoid a copy as the implementation will be more complicated.